3.rb

```diff
@@ -0,0 +1,7 @@
+def caps(name)
+    return name.upcase if name.length > 10
+    return name
+end
```

Comment:
return keyword is not necessary in the last line
```suggestion
def caps(name)
    return name.upcase if name.length > 10
    name
end
```